templates/cicd/buildspec.yml

```diff
@@ -20,29 +20,29 @@ phases:
     commands:
       - ls -l
       - export COLOR="false"
-      # Find all the local applications in the workspace.
-      - apps=$(./ecs-preview app ls --local --json | jq '.applications[].name' | sed 's/"//g')
+      # Find all the local services in the workspace.
+      - svcs=$(./ecs-preview svc ls --local --json | jq '.services[].name' | sed 's/"//g')
```

Comment:
have we renamed the binary itself yet? not sure if this should say `copilot` instead of `ecs-preview`

Reply:
We haven't yet! I was thinking of updating the Makefile after resolving the billions of merge conflicts with master